prog/postgres/postgres_resource_nexus.rb

```diff
@@ -161,9 +158,9 @@ def before_run
   end
 
   label def wait
-    # Create missing standbys
-    (postgres_resource.required_standby_count + 1 - postgres_resource.servers.count).times do
-      Prog::Postgres::PostgresServerNexus.assemble(resource_id: postgres_resource.id, timeline_id: postgres_resource.timeline.id, timeline_access: "fetch", private_subnet_id: representative_server.vm.private_subnets.first.id)
+    # Only create one standby at a time to ensure that they are allocated on different hosts
+    if postgres_resource.required_standby_count + 1 > servers.count && servers.none? { _1.vm.vm_host.nil? }
+      Prog::Postgres::PostgresServerNexus.assemble(resource_id: postgres_resource.id, timeline_id: postgres_resource.timeline.id, timeline_access: "fetch", private_subnet_id: representative_server.vm.private_subnets.first.id, exclude_host_ids: servers.map { _1.vm.vm_host.id })
```

Comment:
Should we add an exclusion for the development? If we only have one host, can we still provision a DB with HA?